src/popup/epics/peerjsEpic.ts

```diff
@@ -82,6 +87,8 @@ const peerjsOpenEpic = (
   action$.pipe(
     ofType(actions.STREAM_MSG),
     map(msg => msg.payload),
+    ofType(EpicMessageOrigin.BG),
```

Comment:
If RTC_* is wrapped by EpicMessageOrigin.BG, then the wrap of STREAM_MSG is not necessary
otherwise the order of unwrapping is essential, could bring bugs.

Reply:
Okay, will check it.